core-web/libs/portlets/dot-analytics/data-access/src/lib/services/dot-analytics.service.ts

```diff
@@ -0,0 +1,148 @@
+import { Observable } from 'rxjs';
+
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+
+import { map } from 'rxjs/operators';
+
+import { createCubeQuery } from '../utils/cube/cube-query-builder.util';
+
+import type {
+    AnalyticsApiResponse,
+    Granularity,
+    PageViewDeviceBrowsersEntity,
+    PageViewTimeLineEntity,
+    TimeRange,
+    TopPagePerformanceEntity,
+    TopPerformaceTableEntity,
+    TotalPageViewsEntity,
+    UniqueVisitorsEntity
+} from '../../index';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DotAnalyticsService {
+    #BASE_URL = '/api/v1/analytics/content/_query/cube';
+    #http = inject(HttpClient);
+
+    /**
+     * Total de pageviews en el período especificado
+     */
+    totalPageViews(
+        timeRange: TimeRange = 'from 7 days ago to now'
```

Comment:
Maybe we could use `TimeRange.LAST_7_DAYS`

Reply:
I did this better
```
/**
 * Default time range for analytics queries.
 */
export const DEFAULT_TIME_RANGE = TimeRange.LAST_7_DAYS;
```

instead of exposing the const.